src/indexer/merger.rs

```diff
@@ -201,6 +201,8 @@ impl IndexMerger {
             }
             merged_doc_id_map.push(segment_local_map);
         }
+
+        let mut field = Field(u32::max_value());
```

Comment:
I'll change it for an Option<>

Right now `u32::max_value()` is here to make sure the first field is added.